.github/workflows/support-files/notifications/send_message.js

```diff
@@ -38,6 +38,11 @@ function validateContext() {
       'Paperkey is not defined. Please set env var KEYBASE_NYMBOT_PAPERKEY',
     );
   }
+  if (!context.env.KEYBASE_NYMBOT_TEAM) {
+    throw new Error(
+      'Team is not defined. Please set env var KEYBASE_NYMBOT_TEAM',
+    );
+  }
```

Comment:
All the current GitHub Actions will fail, because they do not set this environment variable. So this check is a very bad idea.